tests/test_completion_core.py

```diff
@@ -32,141 +32,3 @@ def test_dict_response(self):
     def test_incomplete_dict_response(self):
         self.assertEqual(parse_completion_response({'items': [], 'isIncomplete': True}), ([], True))
 
-
-class CompletionFormattingTests(unittest.TestCase):
-
-    def test_only_label_item(self):
-        result = format_completion({"label": "asdf"})
-        self.assertEqual(len(result), 2)
-        self.assertEqual("asdf", result[0])
-        self.assertEqual("asdf", result[1])
-
-    def test_prefers_insert_text(self):
-        result = format_completion({"label": "asdf", "insertText": "Asdf"})
-        self.assertEqual(len(result), 2)
-        self.assertEqual("asdf", result[0])
-        self.assertEqual("Asdf", result[1])
-
-    def test_ignores_text_edit(self):
-
-        # partial completion from cursor (instead of full word) causes issues.
-        item = {
-            'insertText': '$true',
-            'label': 'true',
-            'textEdit': {
-                'newText': 'rue',
-                'range': {
-                    'start': {'line': 0, 'character': 2},
-                    'end': {'line': 0, 'character': 2}
-                }
-            }
-        }
-
-        result = format_completion(item)
-        self.assertEqual(len(result), 2)
-        self.assertEqual("$true", result[0])
-        self.assertEqual("\\$true", result[1])
-
-    def test_use_label_as_is(self):
-        # issue #368
-        item = {
-            'insertTextFormat': 2,
-            # insertText is present, but we should prefer textEdit instead.
-            'insertText': 'const',
-            'sortText': '3f800000const',
-            'kind': 14,
-            # Note the extra space here.
-            'label': ' const',
-            'textEdit': {
-                'newText': 'const',
-                'range': {
-                    # Replace the single character that triggered the completion request.
-                    'end': {'character': 13, 'line': 6},
-                    'start': {'character': 12, 'line': 6}
-                }
-            }
-        }
-        result = format_completion(item)
-        self.assertEqual(result, ('const\t  Keyword', 'const'))
-
-    def test_text_edit_intelephense(self):
```

Comment:
I think we can just remove these tests:
- test_text_edit_intelephense
- test_text_edit_clangd
- test_missing_text_edit_but_we_do_have_insert_text_for_pyls 
because they are no longer relevant.